testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_custom_conflicts.py

```diff
@@ -665,7 +665,7 @@ def test_non_blocking_custom_conflicts_resolution(params_from_base_test_setup, r
             new_docs_body[doc_id] = [data]
             data = property_updater(data)
             random_value = random_string(length=10, printable=True)
-            data["update_during_CCR"] = random_value
+            data["update_during_CCR"] = random_value + '"'
```

Comment:
is this for your testing only? I don't see a reason to append a double quote to the end of the random value of string.

Reply:
removed ,used it to make sure all aways random string has cote.